packages/gluestick/src/config/__tests__/__snapshots__/compileWebpackConfig.test.js.snap

```diff
@@ -232,6 +238,34 @@ Object {
               "test": /\\\\\\.\\(scss\\|css\\)\\$/,
             },
           },
+          NormalModuleReplacementPlugin {
```

Comment:
ChunksPlugin is really weird and requires a Webpack config to be passed into it. I'm glad we're getting rid of this soon.

Looking through the universal-webpack code, supplying extra plugins shouldn't do anything.